index.html

```diff
@@ -0,0 +1,189 @@
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/4f02d055e5.js"
+      crossorigin="anonymous"
+    ></script>
+    <script src="https://kit.fontawesome.com/870e4168d3.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css" />
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <script src="script.js" defer></script>
+  </head>
+
+  <body>
+    <header>
+      <div class="top-bar">
+        <ul class="top-bar-ul">
+          <li><i class="fab fa-facebook-f"></i></li>
+          <li><i class="fab fa-twitter"></i></li>
+          <li>English</li>
+          <li>Mypage</li>
+          <li>Logout</li>
+        </ul>
+      </div>
+      <div class="header-bar">
+        <div class="logo">
+          <a href="./index.html"
+            ><div class="logo-climate">
+              <i class="fa-duotone fa-hand-holding-seedling"></i>
+            <p>SAVE THE PLANET </p>
+            <!-- <p>IG</p> -->
+          </div></a
+          >
+          <div class="close-btn"><i class="fas fa-times fa-2x"></i></div>
+        </div>
+        <nav>
+          <ul class="nav-ul">
+            <li><a class="menu-link" href="./about.html">About</a></li>
+            <li><a class="menu-link" href="#program">Program</a></li>
+            <li><a class="menu-link" href="#">Join</a></li>
+            <li><a class="menu-link" href="#partner">Sponsors</a></li>
+            <li><a class="menu-link" href="#">News</a></li>
+          </ul>
+        </nav>
+        <button class="campaign">Campaign</button>
+      </div>
+    </header>
+    <main>
+      <section class="intro">
+        <div class="menu">
+          <i class="fas fa-bars fa-2x"></i>
+        </div>
+        <h2>"LET U ALL HELP SAVE OUR CLIMATE"</h2>
+        <h1>GLOBAL CLIMATE FIXING CONFERENCE 2022</h1>
+        <div class="intro-descr">
+          <p>
+            This is the official page of  the World Climate Fixing Conference
+            which is held annually. This conference is held tech company that 
+            are solving the world problems of climate condition and change.
+          </p>
+        </div>
+        <div class="date-local">
+          <h3>2022.03.016(WED) ~ 11(FRI)</h3>
+          <h4>Silicon Valley, Carlifornia, U.S.A</h4>
+        </div>
+      </section>
+      <section class="main-program" id="program">
+        <h2>Main Program</h2>
+        <hr />
+        <div class="programs">
+          <div class="program">
+            <img
+              class="program-icon"
+              src="./images/lecturers.png"
+              alt="lecture"
+            />
+            <h3 class="program-title">Lecture</h3>
+            <p class="program-descr">
+              Listen to the expert from various diverse locations on climate solutions.
+            </p>
+          </div>
```

Comment:
- [ ] Please kindly transform the `<div class="program">` cards into a more semantic element like an `<article>` to make the code cleaner and easier to navigate and read.
- [ ] I also suggest adding a hover state to the program cards as seen in the original design

![image](https://user-images.githubusercontent.com/4247740/159019213-eb51fa8b-0fe8-4f8e-91e5-f0b42ca21fe6.png)

Reply:
- [ ] As the the previous reviewer stated,  kindly change `<div class="program">` to `<section>` tag to improve the HTML semantics.
- [ ] Kindly add the hover to the program cards.